x-pack/plugins/apm/server/lib/anomaly_detection/create_anomaly_detection_jobs.ts

```diff
@@ -101,6 +101,11 @@ async function createAnomalyDetectionJob({
       startDatafeed: true,
       jobOverrides: [
         {
+          datafeed_config: {
```

Comment:
This could actually go straight into the datafeed config of the apm module, in https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/ml/datafeed_high_mean_transaction_duration.json, which saves you having to override it at the code level.